tools/plugins/com.liferay.ide.project.core/build.properties

```diff
@@ -7,4 +7,5 @@ bin.includes = META-INF/,\
                schema/,\
                OSGI-INF/,\
                lib/jackson-core-asl-1.9.13.jar,\
-               lib/jackson-mapper-asl-1.9.13.jar
+               lib/jackson-mapper-asl-1.9.13.jar,\
+               liferay-extension-point/liferay-extension-point.json
```

Comment:
Lets call this file "services-static.json" and you can put it at the root of the budnel